kedro_neptune/config.py

```diff
@@ -48,7 +48,7 @@ def get_neptune_config() -> NeptuneConfig:
     project = _parse_config_input(config['neptune']['project'])
     base_namespace = config['neptune']['base_namespace']
     source_files = config['neptune']['upload_source_files']
-    enabled = config['neptune'].get('enabled', True)
+    enabled = bool(_parse_config_input(config['neptune']))
```

Comment:
Hey @guemues, I think it's a little bit too short :wink:. At least you should point to expected configuration key (`enabled`) instead of a section, so this should be something like `bool(_parse_config_input(config['neptune'].get('enabled', True))`

Reply:
Remember that `enabled` is only an optional key :wink:

Reply:
Thanks Raalsky, I tried to keep it as simple as possible but it turns out to be too simpler :)